SU-HSDO/suhumsci
# SHS-5497: respect reduced motion for nav caret animation

# READY FOR REVIEW

## Summary
Disable caret animation in the nav on page load or when clicking on a top-level nav item to expand a submenu, when the `Reduce Motion` setting is enabled.

## Need Review By (Date)
3/13

## Urgency
low

## Steps to Test
1. Go to any page
2. Check the nav caret animation in all the kind of navigation (main nav, sidebar nav)
3. On your computer, go to System Settings > Accessibility > Display and turn on the `Reduce motion` setting
4. Confirm there's no nav caret animation

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
